networks/meld/tokens/0x333002599c1bad8d3861baa959d0f9e2a4d57c06.json

```diff
@@ -7,8 +7,8 @@
   "tokenAddress": "0x333002599c1bad8d3861baa959d0f9e2a4d57c06",
   "tokenColorTheme": 4,
   "tokenIcon": "/static/tokens/USDT.svg",
-  "tokenName": "Tether USD",
-  "tokenTicker": "USDT",
+  "tokenName": "Chainport USDT",
+  "tokenTicker": "wUSDT",
```

Comment:
revert this change (https://meldscan.io/token/0x333002599C1bad8d3861bAa959d0f9E2a4d57C06)